### ISSUE TITLE
iOS - Home Screen - VoiceOver is Higlighting Icons

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#122]
When reading the home screen with VoiceOver, all the icons next to the text in the main content section are now being highlighted and displayed on the screen. The issue is new and hadn't shown up in the previous version of the application. I think it has to do with the way the links/buttons are created or that these now might be SVG icons. Look for odd black outlines on screen.

Screenshot from our iPad.

Similar issues happen on other screens, which I will create issues.

![comply chain - home screen - Step 1 icon highlighted large in black when using voiceover](https://user-images.githubusercontent.com/10423487/74768301-53834b80-5256-11ea-8412-1bfc967825bd.png)

**Comment by jfc3-dol** [comment]
Adding two more screenshots from our iPhone 7.

**Not sure where the icon is from anymore.**

![comply chain - home screen - ios - large icon showing on iphone 7](https://user-images.githubusercontent.com/10423487/74769025-9bef3900-5257-11ea-9770-92cffc556271.png)

**Search icon from footer.**

![comply chain - home screen - ios - large search from footer showing on iphone 7](https://user-images.githubusercontent.com/10423487/74769027-9bef3900-5257-11ea-87aa-140666bc1068.png)

**Comment by harpreetsdev** [comment]
@jfc3-dol 

This one is fixed. Icons are no longer being highlighted in VO mode. Please review and let us know.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

Thanks, I had planned to check this issue because it was marked as fixed in the spreadsheet I was sent. I was planning to get to it today, if possible.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

This one was fixed Sprint 9. You should have the fixed version in the build from last sprint. Please let me know if you find it or I'll send you an update notification from test flight.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

Thanks, I will do.

I'm starting on the iOS issues in a short bit from the top and moving my way down the spreadsheet.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

The icons are no longer getting highlighted when using  VoiceOver to read the page, so I'm going to close the issue.
